azure-pipelines.yml

```diff
@@ -95,24 +96,48 @@ stages:
           secureFile: SonarSourceSecret.snk
 
       - task: DownloadSecureFile@1
-        displayName: 'Download pfx file'
-        name: pfx
+        displayName: 'Download p12 file'
+        name: SM_CLIENT_CERT
         inputs:
```

Comment:
Add a comment that this file is used by the `DigiCert Signing Manager KSP` Key Storage Provider to authenticate against the DigiCert private key provider server.